integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/AbstractTomcatIT.java

```diff
@@ -33,7 +33,7 @@ public AbstractTomcatIT(final String tomcatVersion) {
 
     @Override
     protected void enableDebugging(GenericContainer<?> servletContainer) {
-        servletContainer.withEnv("CATALINA_OPTS", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005");
+        servletContainer.withEnv("CATALINA_OPTS", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005");
```

Comment:
is `suspend=y` right?

Reply:
Since debugging is disabled by default (through `co.elastic.apm.servlet.AbstractServletContainerIntegrationTest#ENABLE_DEBUGGING`), this is not super crucial. I think that I use `suspend=y` more often than not, but let's keep `n` as the default.